README.md

```diff
@@ -4,12 +4,25 @@
 
 _The project is a part of the course TDT4290 Customer driven project in the autumn of 2019._
 
+![GIF of the finished product in x4 speed](assets/demo/aqueous_demo_4x.gif)
+_GIF of the finished product in x4 speed_
+
 During this project, we will build a graphical user interface for controlling a [ROV](https://en.wikipedia.org/wiki/Remotely_operated_underwater_vehicle) with an XBOX controller built on web technologies ([Electron](https://electronjs.org/), Node, [React](https://reactjs.org/)).
 
 The GUI will incorporate a videofeed, ability to connect to a ROV simulation and use a subset of the [IMC protocol](https://www.lsts.pt/toolchain/imc) for communication.
 
 ###### This project is a continuation of a customer-driven project from last year, which can be found [here](https://github.com/Kpro11/Aqeous), however rewritten with web technologies.
 
+## Installation
+
+If you just want to run the project, go to the [release tab](https://github.com/eirikdahlen/TDT4290_Aqueous/releases) in this GitHub repository and it should be plug and play.
+
+Every release of the product is available here as an installable `.exe` file (for Windows), along with the features included in each release.
+
+Simply download the `AqueousInstaller.exe` file, run as Administrator and run the program as any other desktop application.
+
+If you are on any other operation system you will have to build the project your self. It is eary (se below).
```

Comment:
eary?

Reply:
se?

Reply:
Typos:
* your self = yourself
* eary = easy?
* se = see
Maybe rephrase last part to "See section .."